contracts/interfaces/IHATVaultsRegistry.sol

```diff
@@ -284,6 +285,7 @@ interface IHATVaultsRegistry {
         address indexed _asset,
         address indexed _owner,
         address _committee,
+        string _symbol,
```

Comment:
why not add `_name` as well?

Reply:
Well I didn't know if I should add it or derive from the symbol.